fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/ReindexResourceDAO.java

```diff
@@ -294,6 +296,33 @@ public void updateParameters(String tablePrefix, List<ExtractedParameterValue> p
                 throw translator.translate(x);
             }
         }
+
+        // Update the parameter hash in the LOGICAL_RESOURCES table
+        updateParameterHash(connection, logicalResourceId, parameterHashB64);
+
         logger.exiting(CLASSNAME, METHODNAME);
     }
+
+    /**
+     * Updates the parameter hash in the LOGICAL_RESOURCES table.
+     * @param conn the connection
+     * @param logicalResourceId the logical resource ID
+     * @param parameterHashB64 the Base64 encoded SHA-256 hash of parameters
+     * @throws SQLException
+     */
+    protected void updateParameterHash(Connection conn, long logicalResourceId, String parameterHashB64) throws SQLException {
+        final String SQL = "UPDATE logical_resources SET parameter_hash = ? WHERE logical_resource_id = ?";
+        try (PreparedStatement stmt = conn.prepareStatement(SQL)) {
+            // bind parameters
+            stmt.setString(1, parameterHashB64);
+            stmt.setLong(2, logicalResourceId);
+            stmt.executeUpdate();
+            if (logger.isLoggable(Level.FINEST)) {
+                logger.finest("Update parameter_hash [" + parameterHashB64 + "] for logicalResourceId [" + logicalResourceId + "]");
```

Comment:
Might be useful to add [took = xx ms] timing information for this update. This would make it easier to see if this update slows down over time (due to the effect of tombstones in PostgreSQL, until vacuum cleans things up). I don't think it will (because there are no indexes being changed here)...but individual timing in the finest message would be helpful.

Reply:
Updated.